docs/src/routes/checkbox/index.js

```diff
@@ -62,39 +62,74 @@ export default class CheckboxPage extends Component {
         </div>
 
         <div className="mdc-typography--display1">Demo </div>
+
+        <div className="mdc-typography--title">Basic Checkbox </div>
         <Formfield>
-          <Checkbox
-            id="basic-checkbox"
-            ref={cb => {
-              this.cb = cb;
-            }}
-          />
-          <label for="basic-checkbox" id="basic-checkbox-label">
-            This is my checkbox
-          </label>
+          <Checkbox id="basic-checkbox" />
+          <label for="basic-checkbox">Basic checkbox</label>
         </Formfield>
-        <Button
-          raised={true}
-          onClick={() => {
-            this.cb.MDComponent.indeterminate = true;
-          }}
-        >
-          Make indeterminate
-        </Button>
+        <CodeBlock>
+          <code class="lang-html">
+            {`
+<Formfield>
```

Comment:
can we extract these into a separate file? cuz `<pre>` tag is white space sensitive

Reply:
Sure, no problem